Change service name to match change in runner repo

donoftime opened the pull request:
I missed this name change because I had a stale cache of the
gitlab-ci-multi-runner. As of the below commit, ayufan changed the
default service name to be gitlab-runner instead of the longer
gitlab-ci-multi-runner:

https://github.com/ayufan/gitlab-ci-multi-runner/commit/cba7ffaa69e61b329c3d02808537eefdfbc747ba

donoftime committed:
Change service name to match change in runner repo

I missed this name change because I had a stale cache of the
gitlab-ci-multi-runner. As of this commit, ayufan changed the
default service name to be gitlab-runner instead of the longer
gitlab-ci-multi-runner:

https://github.com/ayufan/gitlab-ci-multi-runner/commit/cba7ffaa69e61b329c3d02808537eefdfbc747ba

frankiethekneeman commented:
@donoftime Thanks for all MRs!

I believe the name change is only valid _after_ version 0.4.2-1 - which this package is still using as the default installation version.

Additionally, if the service name is going to change, then the 'Ensure Service' command should probably use the updated binary name `gitlab-runner`

donoftime commented:
Yeah, I think you're right. I'll throw up an improved commit in a bit to address that.

donoftime committed:
Flip default so newer releases will use latest packages

Having the default be used for the older releases (5.0 and 6.0) meant
that in the future we would have to continue to update to keep up.
By flipping the default, the only time we need special processing
is for older releases which likely won't be changing anyway.

I also changed the service name to be dependent on the runner release
we are using.

donoftime commented:
I've made the change and tested with Centos 6.4 and OL7, and both apply cleanly.

donoftime commented:
Annddd.....clean on ubuntu 14.04.

donoftime committed:
Use new binary name for later runner releases

The binary name changed from gitlab-ci-multi-runner to just
gitlab-runner. There is a symlink for the old name still around,
but it will likely go away in the future so we should switch to
use the new binary name.

donoftime committed:
Correct repoLocation for debian systems

I noticed that the Add Repo was running every time I applied, and
it is due to me being overzealous on my initial switch from apt
to deb. Looks like that one is actually in the apt directory.

donoftime commented:
Verified again on all three with the new binary name and fixed debian repoLocation.

donoftime commented:
Please publish to the forge when you get a chance so I can use it :smile:

frankiethekneeman commented:
@donoftime Just did it - Sorry, I manage this repo from work, and was out of the office yesterday.

donoftime commented:
No worries, thanks for the update :+1:
